.github/workflows/ci.yml

```diff
@@ -154,8 +154,8 @@ jobs:
       - name: VSCode extension lint
         working-directory: extensions/vscode
         run: pnpm run lint
-      - name: VSCode extension pnpm build
-        run: just build-extension
+      - name: VSCode extension pnpm build and package
+        run: just package-extension
```

Comment:
@alecthomas this should catch the error in CI now. It was just building before and not actually packaging. I'm going to avoid testing the `publish` in CI for now as that would be way more complex.